apm-sniffer/apm-sdk-plugin/spring-plugins/webflux-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/spring/webflux/v5/StatusInterceptor.java

```diff
@@ -29,10 +29,16 @@
     @Override
     public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allArguments, Class<?>[] argumentsTypes,
         MethodInterceptResult result) throws Throwable {
-        HttpResponseStatus status = (HttpResponseStatus)allArguments[0];
-        if (status.code() > 400) {
+        int code;
+        if (allArguments[0] instanceof Integer) {
```

Comment:
The `HttpServerOperations21xInstrumentation.java` and `HttpServerOperations20xInstrumentation.java` had checked that the type of the first parameter is `HttpResponseStatus` and why need to check the arguments type again?